### ISSUE TITLE
Benchmark : CNN with 3 Conv Layers - Accuracy 92.1% on FashionMNIST and 99.2% on MNIST

### ISSUE THREAD

**Issue by GunjanChhablani** [zalandoresearch/fashion-mnist#116]
The model details are as follows:
- No Preprocessing
- The dataset is one-hot encoded 
- Trained using BCE Loss over a Softmax Output
- The parameter initialization is Xavier Normal
- The layers in sequence are:

1. Convolutional layer with 16 feature maps of size 3 x 3
2. BatchNorm layer followed by Swish activation.
3. Max Pooling layer of size 2 x 2.
4. Convolutional layer with 32 feature maps of size 3 x 3
5. BatchNorm layer followed by Swish activation.
6. Max Pooling layer of size 2 x 2.
7. Convolutional layer with 64 feature maps of size 3 x 3
8. BatchNorm layer followed by Swish activation.
9. Max Pooling layer of size 2 x 2.
10. Fully connected layer of size 10.
11. Softmax Layer of size 10.

Accuracy achieved on Fashion MNIST Test Dataset is 92.1 %
Accuracy achieved on MNIST Test Dataset is 99.2% .
This network has been implemented in PyTorch. It uses the recent activation function - Swish which has been implemented in the code. The code can be found [here](https://github.com/GunjanChhablani/CNN-with-FashionMNIST).
